AWS.md

```diff
@@ -1,7 +1,21 @@
 General
 -------
 
- * Use CloudFormation or ElasticBeanStalk to define infrastructure in AWS.
+ * Provision and manage all AWS resources using infrastructure as code
+   * Prefer to use [CDK](https://github.com/guardian/cdk) to generate CloudFormation. You might find that older projects still use CloudFormation directly; these should be migrated to CDK where possible.
+   * Prefer to [import resources](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/resource-import.html) into CFN over manually configuring them
+ * Tag resources with:
+   * `Stack` - the broad umbrella the service sits under e.g. `deployTools`
+   * `Stage` - the environment, typical values are:
+     * `PROD` for production
+     * `CODE` for pre-production
+     * `INFRA` for infrastructure or singleton resources e.g. [elasticsearch-node-rotation](https://github.com/guardian/elasticsearch-node-rotation)
+   * `App` - the individual service e.g. `prism`
+ * If a resource needs to be shared across multiple stacks, prefer to define it in it's own CFN template with stage `INFRA`
```

Comment:
I worry that this line might confuse people. For example, if there is a stream or queue which links together two accounts or stacks, it probably makes sense for this to have a `CODE` and `PROD` version, rather than a single `INFRA` stack. 

That said, I'm struggling to come up with a clear statement which explains when to use `INFRA` - I would hope that DevX are the only people who need to use this stage. We often seem to use it when it isn't possible (or doesn't quite make sense) to have a pre-production environment!

Reply:
I agree. What about dropping the bit about stage entirely?

```
If a resource needs to be shared across multiple environments, prefer to define it in it's own CFN template for clarity as a resource cannot be defined in multiple templates.
```

Reply:
Yes, that looks good to me 👍